gammapy/scripts/tests/test_image_pipe.py

```diff
@@ -0,0 +1,76 @@
+"""Example how to make an acceptance curve and background model image.
+"""
+from astropy.table import Table
+import astropy.units as u
+from astropy.io import fits
+from astropy.coordinates import SkyCoord, Angle
+from astropy.units import Quantity
+from gammapy.datasets import gammapy_extra
+from gammapy.background import EnergyOffsetBackgroundModel
+from gammapy.utils.energy import EnergyBounds, Energy
+from gammapy.data import DataStore
+from gammapy.utils.axis import sqrt_space
+from gammapy.image import bin_events_in_image, disk_correlate, SkyMap, ExclusionMask
+from gammapy.background import fill_acceptance_image
+from gammapy.region import SkyCircleRegion
+from gammapy.stats import significance
+from gammapy.background import OffDataBackgroundMaker
+from gammapy.scripts import ImageAnalysis
+
+
+
+
```

Comment:
But this test requires `gammapy-extra`, right? So just add `@requires_data('gammapy-extra')`. This might fix the Travis-CI build as well.